packages/toolpad-app/src/server/data.ts

```diff
@@ -455,3 +461,43 @@ export async function loadDom(appId: string, version: VersionOrPreview = 'previe
 export async function loadRenderTree(appId: string, version: VersionOrPreview = 'preview') {
   return appDom.createRenderTree(await loadDom(appId, version));
 }
+
+const LATEST_RELEASE_API_URL = 'https://api.github.com/repos/mui/mui-toolpad/releases/latest';
```

Comment:
Let's create a separate file for this `updateCheck`. And let's move everything that's below here inside of it?

Reply:
Makes sense. 
`getLatestRelease` as a filename feels clearer to me. What do you think?